invenio_records_rest/links.py

```diff
@@ -2,6 +2,9 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2015, 2016 CERN.
+# Copyright (C) 2017 Swiss Data Science Center (SDSC)
```

Comment:
FYI: We will wait with the integration until we've implemented the license change of the repository GPL to MIT to avoid further complicating the license change (basically integrating now, would mean we would have to ask SDSC for permission for the change from GPL to MIT). Sorry for this, we will try to attack this repo as one of the first to speed it up.

Reply:
I can send you written permission if you need one.

Reply:
The problem is fully on the CERN-end. We're awaiting answer from CERN KT on how we need to obtain permissions from third-party in a legally correct way. I'll try to make sure the PR goes in before the CERN annual closure.